CHANGES.rst

```diff
@@ -19,6 +19,38 @@ jplspec
 
 - minor improvement to lookuptable behavior [#3173,#2901]
 
+mocserver
+^^^^^^^^^
+
+- Switch to https instead of http for the default url (allows pyodide to use the 
+  module) [#3139]
+
+- Add ``TimeMOC`` and ``STMOC`` as possible entries in ``MOCServer.query_region`` to
+  allow temporal and space-time searches [#3139]
+
+- ``return_moc`` now allows to ask for a Time-MOC or a Space-Time MOC rather than only
+  Space-MOCs [#3139]
+
+- Fix query by MOC that would write a file ``moc.fits`` where the method was executed in
+  overwriting mode (potentially deleting data if there was a conflicting file) [#3139]
+
+- [:warning: BREAKING] Returned tables now have a default list of fields instead of the
```

Comment:
This warning won't work here, I'll edit this out during rebase.

Reply:
actually, I leave this to the before-release changelog cleanup, and go ahead with the PR as is.

Thanks @ManonMarchand!